packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -175,6 +173,14 @@ class TransactionService {
       }
     }
 
+    /**
+     * For unsampled transactions, avoid capturing various timing informations
+     * as spans as they are dropped anyways before sending to the server
```

Comment:
nit: you should use 'information' instead of 'informations',  also 'as they' is redundant.